platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -135,72 +128,78 @@ public SystemPropertiesAdmin(GuestClaimsHandlerExt guestClaimsHandlerExt)
 
     ArrayList<SystemPropertyDetails> properties = new ArrayList<>();
 
-    properties.add(
-        getSystemPropertyDetails(
-            SystemBaseUrl.EXTERNAL_HOST,
-            EXTERNAL_HOST_TITLE,
-            EXTERNAL_HOST_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemBaseUrl.EXTERNAL_HTTP_PORT,
-            EXTERNAL_HTTP_PORT_TITLE,
-            EXTERNAL_HTTP_PORT_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemBaseUrl.EXTERNAL_HTTPS_PORT,
-            EXTERNAL_HTTPS_PORT_TITLE,
-            EXTERNAL_HTTPS_PORT_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemBaseUrl.INTERNAL_HOST,
-            INTERNAL_HOST_TITLE,
-            INTERNAL_HOST_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemBaseUrl.INTERNAL_HTTP_PORT,
-            INTERNAL_HTTP_PORT_TITLE,
-            INTERNAL_HTTP_PORT_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemBaseUrl.INTERNAL_HTTPS_PORT,
-            INTERNAL_HTTPS_PORT_TITLE,
-            INTERNAL_HTTPS_PORT_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemInfo.ORGANIZATION,
-            ORGANIZATION_TITLE,
-            ORGANIZATION_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemInfo.SITE_CONTACT,
-            SITE_CONTACT_TITLE,
-            SITE_CONTACT_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemInfo.SITE_NAME,
-            SITE_NAME_TITLE,
-            SITE_NAME_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemInfo.VERSION, VERSION_TITLE, VERSION_DESCRIPTION, null, systemDotProperties));
+    Properties systemDotProperties = null;
+    try {
+      systemDotProperties = new Properties(systemPropertiesFile);
```

Comment:
✏️ Not a blocker for this PR: I don't like that we have to read this file twice. Once when we populate the installer page and then another time when we save the installer page. Are we worried about the system.properties being changed in between those 2 reads? Can we rely on the system properties instead to populate the installer page and only ready the system.properties file in order to overwrite the few values. Or, can we use the same read for the corresponding write?